.env.development

```diff
@@ -18,6 +18,7 @@ SITE_NAME=localhost
 MARKETING_SITE_BASE_URL=http://localhost:18000
 SUPPORT_URL=http://localhost:18000/support
 ORDER_HISTORY_URL=http://localhost:8140/orders/unified/order_history/
+RECEIPT_URL=http://localhost:8140/orders/unified/order_receipt
```

Comment:
```suggestion
RECEIPT_URL=http://localhost:8140/orders/unified/order_receipt/
```

Reply:
I still am chaffing at this... whats the logic?

Reply:
Also that is here: https://github.com/edx/edx-internal/pull/10177